zkevm-circuits/src/super_circuit.rs

```diff
@@ -184,6 +189,8 @@ impl SubCircuitConfig<Fr> for SuperCircuitConfig<Fr> {
         log_circuit_info(meta, "poseidon table");
 
         let bytecode_table = BytecodeTable::construct(meta);
+        let bytecode_table1 = BytecodeTable::construct(meta);
```

Comment:
Need to feature gate this:
```suggestion
        #[cfg(feature = "dual-bytecode")]
        let bytecode_table1 = BytecodeTable::construct(meta);
```